spec/diddocuments.md

```diff
@@ -500,3 +492,18 @@ Resulting DID document:
   ]
 }
 ```
+
+### Basic KERI event details
+[DID Document from KERI Events](#did-document-from-keri-events) introduced the core [[ref: KERI event stream]] and related DID Document concepts. This section provides additional details regarding the basic types of KERI events and how they relate to the DID document.
+
+#### Key state events
+When processing the [[ref: KERI event stream]] there are two broad types of key state events (KERI parlance is 'establishment events') that can alter the key state of the DID document.  Every [[ref: AID]] starts with an [[ref: inception event]]. If that inception event does not commit to a future set of rotation key hashes, then the AID is non-transferrable because the identifier can never change key state. If the inception event does commit to a future set of rotation key hashes, then the AID is transferrable and there could be future rotation events that transfer the key state from the current keys to the rotation keys that were committed to in the previous key state event.
```

Comment:
I think this should be “that can alter the key state of the AID.” The DIDDoc derives from the key state, but the DIDDoc doesn’t have key state.

Reply:
I think the bit about the transferrable, non-transferrable should be in the inception event bullet point, not in the title.

I assume we can’t change this, but the term “transferrable” I find to be quite confusing.  It means AFAIK that the the keys in the DID can’t be rotated.  I’m sure there is a reason behind the term “transfer”...

Reply:
> I think this should be “that can alter the key state of the AID.” The DIDDoc derives from the key state, but the DIDDoc doesn’t have key state.

I've fixed this wording and it will be in the next PR.

Reply:
> I think the bit about the transferrable, non-transferrable should be in the inception event bullet point, not in the title.
> 
> I assume we can’t change this, but the term “transferrable” I find to be quite confusing. It means AFAIK that the the keys in the DID can’t be rotated. I’m sure there is a reason behind the term “transfer”...

The term "transferrable" means that control can be transferred from the current keys to the next keys that were committed to (via the rules of pre-rotation). Both inception and rotation events can provide transferability by specifying a set of next key hashes that commit to the transferability. It is also possible to end the ability to transfer control to next keys by not committing to a set of next keys. This could happen in the inception or rotation event. So perhaps I should keep it in this section above but be more explicit that it applies to both inception and rotation?

Reply:
I’ve long heard in the broader community and used the term "rotate keys" for the operation we’re talking about.  And we enable it in KERI by specifying evidence of “pre-rotation” keys.  AFAIK, only the KERI community uses the “Transferrable” — which for me has no connection to key rotation.  When I hear it, I jump to transferring control of the DID from one entity to another, or perhaps transfering a DID from one ledger (e.g. DID identifier) to another.

Reply:
Agree that the pre-rotation mechanism must be included for the inception event and rotation events to enable additional rotations.